tests/markdown_link/title.md

```diff
@@ -1 +1,26 @@
 [hello](#world "title")
+[hello](#world 'title')
+[hello](#world (title))
+
+[a](https://example.com "\"")
+[a](https://example.com '\"')
+[a](https://example.com (\"))
+
+[a](https://example.com "\'")
+[a](https://example.com '\'')
+[a](https://example.com (\'))
+
+[a](https://example.com "\'")
+[a](https://example.com '\)')
+[a](https://example.com (\)))
+
+<!-- mis-parsing, `\` are missing: -->
+
+[a](https://example.com "\\\"")
+[a](https://example.com '\\\'')
+[a](https://example.com (\\\)))
+
+[a](https://example.com "\\'")
+[a](https://example.com '\\"')
+[a](https://example.com (\\"))
+
```

Comment:
Could you add these test cases https://github.com/prettier/prettier/pull/3481#discussion_r156973153 so as to ensure it won't happen again?

Reply:
⬆️

Reply:
👍 ✅